_posts/2022-9-5-Diplomat @ Google.md

```diff
@@ -155,10 +177,10 @@ impl<'a> Foo<'a> {
     }
 }
 ```
-To be honest, I've never had to use lifetime bounds outside of borrowing
-`Deserialize` impls, and Diplomat doesn't support generic types, so I'm not sure
-why you'd need bounded lifetimes. But they exist, and I didn't want to leave it
-as someone else's problem.
+I've never had to use lifetime bounds outside of borrowing `Deserialize` impls,
```

Comment:
thought: It's fine to say this, but you do know a concrete use case: borrowing iterators :smile: 

Perhaps "Before I worked on this, I never had to ...." and then mention the concrete use case later.

Reply:
Oh I wasn't even aware of that, thanks